docs/FAQ

```diff
@@ -892,14 +892,11 @@ FAQ
   If there is a bug, read the BUGS document first. Then report it as described
   in there.
 
-  4.9 curl cannot authenticate to the server that requires NTLM?
+  4.9 curl cannot authenticate to a server that requires NTLM?
```

Comment:
The TOC in the beginning needs the same update (to make the script for the web able to connect them)

Reply:
Ah, of course, fixed.